src/qml/MapCanvas.qml

```diff
@@ -15,9 +15,9 @@
  *                                                                         *
  ***************************************************************************/
 
-import QtQuick 2.14
-import QtQuick.Controls 2.14
-import QtQml 2.14
+import QtQuick 2.15
```

Comment:
For reviewer: this is needed as the useful dragThreshold property for the DragHandler was only introduced in Qt 5.15 (hence going from .14 to .15).